entries/entry02.md

```diff
@@ -1,8 +1,86 @@
 # Entry 2
```

Comment:
Sources: 1
EDP: 1
Knowledge: 1
Skills: 1
Professionalism: 1

Integrate the links into your writing. Use more markdown!